### ISSUE TITLE
error installing mumax3 on centos 7 with cuda 10.2

### ISSUE THREAD

**Issue by mherkazandjian** [mumax/3#246]
Hi,

I have been trying to compile/install mumax on centos 7

Linux onode12 3.10.0-957.5.1.el7.x86_64 #1 SMP Fri Feb 1 14:54:57 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

using 

Driver Version: 430.30       CUDA Version: 10.2  

and 

go version go1.13.4 linux/amd64

when i execute the command

````bash
CGO_LDFLAGS="-L/apps/sw/cuda/cuda_10.1.168_418.67/lib64/ -g -O2" CGO_CFLAGS="-I/apps/sw/cuda/cuda_10.1.168_418.67/include -g -O2" go install -v 
````

i get the following error:

````bash
# github.com/mumax/3/cuda/curand
../../cuda/curand/generator.go:46:102: identifier "_Ctype_ulonglong" may conflict with identifiers generated by cgo
````
it looks like there is a go version compatibility issue, since this error disappears when
i use downgraded / used go version 1.11

note that i am using golang go from https://golang.org/dl/ not gccgo.

is go 1.13.4 supported? or this error is expected?

Best,

**Comment by JeroenMulkers** [comment]
This issue is solved in 2cf5c9a69.

Try building the latest source, and let me know how it turns out.

**Comment by mherkazandjian** [comment]
Hi @JeroenMulkers 

the build of the master branch worked with the latest version of go.
Tnx,
